### PR TITLE
moved simplified mmr to sora2-common

### PR DESCRIPTION
**Pull request by N1ghtStorm**

### CODE REVIEW

File: node/Cargo.toml [hunk 1]
```diff
@@ -103,7 +103,7 @@ prometheus-endpoint = { package = "substrate-prometheus-endpoint", git = "https:
 beefy-gadget-rpc = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
 mmr-rpc = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
 mmr-gadget = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.38" }
-leaf-provider-rpc = { git = "https://github.com/sora-xor/sora2-common", branch = "polkadot-v0.9.38" } # 'develop' branch after PR 14
+leaf-provider-rpc = { git = "https://github.com/sora-xor/sora2-common", branch = "develop" }
```
Comment (wer1st) [inline node/Cargo.toml:106]: Please change `https://github.com/sora-xor/sora2-common` to `https://github.com/sora-xor/sora2-common.git` in all places

Comment (N1ghtStorm) [reply node/Cargo.toml]: done

File: pallets/trustless-bridge/bridge-inbound-channel/Cargo.toml [hunk 2]
```diff
@@ -31,7 +31,7 @@ permissions = { path = "../../permissions", default-features = false }
 traits = { git = "https://github.com/open-web3-stack/open-runtime-module-library.git", package = "orml-traits", default-features = false }
 technical = { path = "../../technical", default-features = false }
 
-bridge-types = { git = "https://github.com/sora-xor/sora2-common", branch = "polkadot-v0.9.38", default-features = false } # 'develop' branch after PR 14
+bridge-types = { git = "https://github.com/sora-xor/sora2-common", branch = "develop", default-features = false }
```
Comment (wer1st) [inline pallets/trustless-bridge/bridge-inbound-channel/Cargo.toml:34]:
```suggestion
bridge-types = { git = "https://github.com/sora-xor/sora2-common.git", branch = "develop", default-features = false }
```

**Review by wer1st — APPROVED** [review]

**Review by vovac12 — APPROVED** [review]

File: relayer/parachain-gen/build.rs [hunk 3]
```diff
@@ -38,7 +38,7 @@ fn main() {
         .unwrap()
         .join("src/bytes");
 
-    let branch = &std::env::var("PARACHAIN_METADATA_BRANCH").unwrap_or("develop".to_string());
+    let branch = &std::env::var("PARACHAIN_METADATA_BRANCH").unwrap_or("polkadot-v0.9.38".to_string()); // TODO: Change to develop when #82 is done
```
[the remark this answers is not in the document]
Comment (N1ghtStorm) [reply relayer/parachain-gen/build.rs]: Ok, got it

**Pull request merged**